distro/otel_test.go

```diff
@@ -574,30 +611,26 @@ func (coll *collector) Start(t *testing.T) {
 	require.NoError(t, err)
 	coll.Endpoint = ln.Addr().String() // set actual endpoint
 
-	coll.traceService = &collectorTraceServiceServer{
-		requests: make(chan spansExportRequest, 1),
-	}
-	coll.metricsService = &collectorMetricsServiceServer{
-		requests: make(chan metricsExportRequest, 1),
-	}
+	coll.traceService = &collectorTraceServiceServer{}
+	coll.metricsService = &collectorMetricsServiceServer{}
 
 	var opts []grpc.ServerOption
 	if coll.TLS {
 		creds := credentials.NewTLS(serverTLSConfig(t))
 		opts = append(opts, grpc.Creds(creds))
 	}
 
-	srv := grpc.NewServer(opts...)
-	ctpb.RegisterTraceServiceServer(srv, coll.traceService)
-	cmpb.RegisterMetricsServiceServer(srv, coll.metricsService)
+	coll.grpcSrv = grpc.NewServer(opts...)
+	ctpb.RegisterTraceServiceServer(coll.grpcSrv, coll.traceService)
+	cmpb.RegisterMetricsServiceServer(coll.grpcSrv, coll.metricsService)
 	errCh := make(chan error, 1)
 
 	// Serve and then stop during cleanup.
 	t.Cleanup(func() {
-		srv.GracefulStop()
+		coll.grpcSrv.GracefulStop()
```

Comment:
This seems redundant with the `GracefulStop` in the telemetry retrieval methods.

Reply:
not longer valid after https://github.com/signalfx/splunk-otel-go/pull/1848/commits/854a374398ef4a11b52fcaf03a1f9d6bfb9a63d1